.travis.yml

```diff
@@ -14,19 +14,20 @@ cache:
         - $HOME/.composer/cache
 
 before_script:
+    - source .travis/xdebug.sh
+    - xdebug-disable
     - pecl install ast-1.0.4
     - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-dist
 
 script:
-    - vendor/bin/phpunit --coverage-text --coverage-clover=coverage.clover
     - vendor/bin/phan
+    - xdebug-enable
+    - vendor/bin/phpunit --coverage-text --coverage-clover=coverage.clover
 
 after_script:
-    - |
-        if [[ "$TRAVIS_PHP_VERSION" != 'hhvm' ]]; then
-          wget https://scrutinizer-ci.com/ocular.phar
-          php ocular.phar code-coverage:upload --format=php-clover coverage.clover
-        fi
+    - xdebug-disable
+    - wget https://scrutinizer-ci.com/ocular.phar
```

Comment:
I think we have gotten rid of scrutinizer?

Reply:
Just did :+1: